.coveragerc

```diff
@@ -1589,6 +1589,7 @@ omit =
     homeassistant/components/zwave_me/switch.py
     homeassistant/components/switchbee/__init__.py
     homeassistant/components/switchbee/const.py
+    homeassistant/components/switchbee/coordinator.py
```

Comment:
These coverage lines should really be all moved up to line 1215

Reply:
Yes, there are a few mistakes currently in the file, but switchbee entries should be inserted in the correct place.

Reply:
I think it worth making a "sort" PR for all the file

Reply:
See #78447